docs/governance/index.md

```diff
@@ -1,31 +1,127 @@
 ---
 title: Governance
-summary: Hier staat een korte samenvatting over dit onderwerp
+summary: Hulp voor overheden bij de governance (beleid en onderhoud) van hun algoritmes of AI-systemen.
 icon: material/bank
 ---
+# Governance
+Organiseer en onderhoud je algoritmes en AI met effectieve governance. Governance is het beleid van je organisatie voor het verantwoordelijk omgaan met algoritmes en AI-systemen. Leg bijvoorbeeld vast wie waarvoor verantwoordelijk is.
 
-![governance](../afbeeldingen/bouwblokken/governance.jpg "visuele weergave governance"){align="right"; width="50%"}
+## Voldoen aan de regels
+<details>
+<summary>Eisen governance</summary>
```

Comment:
Waarom kiezen we ervoor om te werken met `<details>` en `<summary>`? Dat doen we verder nergens? 

Dit kan bijvoorbeeld ook zoals op de pagina van [maatregelen](https://minbzk.github.io/Algoritmekader/maatregelen/). Of zie de [bijbehorende bronfile](https://github.com/MinBZK/Algoritmekader/edit/main/docs/maatregelen/index.md)

Reply:
Mijn idee is om de kern (de eisen, maatregelen en instrumenten) steeds als 1 pakketje te tonen onder de noemer 'Voldoen aan de regels'. Dan vind je die info snel en herkenbaar terug op elke pagina. De andere kopjes kunnen dan variëren.